src/sentry/static/sentry/app/views/availableWork/workDefinitions.jsx

```diff
@@ -2,71 +2,71 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {t} from 'app/locale';
 import {Panel, PanelBody} from 'app/components/panels';
-import ProcessListItem from 'app/views/taskDefinitions/processListItem';
+import ProcessListItem from './processListItem';
 import LoadingError from 'app/components/loadingError';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import {groupBy, sum} from 'lodash';
 import ClimsTypes from 'app/climsTypes';
-import {taskDefinitionActions} from 'app/redux/actions/taskDefinition';
+import {workDefinitionActions} from 'app/redux/actions/workDefinition';
 
-export class TaskDefinitions extends React.Component {
+export class WorkDefinitions extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
   renderBody() {
     const {
-      taskDefinitions,
-      tasksLoading,
+      workDefinitions,
+      workUnitsLoading,
       errorMessage,
-      getTaskDefinitionList,
+      getWorkDefinitionList,
     } = this.props;
 
     let body;
-    if (tasksLoading) {
+    if (workUnitsLoading) {
       body = this.renderLoading();
     } else if (errorMessage) {
       body = (
         <LoadingError
           message={errorMessage}
-          onRetry={() => getTaskDefinitionList(this.props.organization)}
+          onRetry={() => getWorkDefinitionList(this.props.organization)}
         />
       );
-    } else if (taskDefinitions.length > 0) {
+    } else if (workDefinitions.length > 0) {
       body = this.renderProcesses();
     } else {
       body = this.renderEmpty();
     }
     return body;
   }
 
-  groupTasksByProcess(taskDefinitions) {
+  groupWorkUnitsByProcess(workDefinitions) {
     const ret = [];
-    const grouped = groupBy(taskDefinitions, (x) => x.processDefinitionKey);
-    for (const [processDefinitionKey, tasks] of Object.entries(grouped)) {
+    const grouped = groupBy(workDefinitions, (x) => x.processDefinitionKey);
```

Comment:
I dont follow here. The method name is "group work unit ...", but here work definitions are grouped.